acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/Page.scala

```diff
@@ -70,22 +59,18 @@ abstract class Page[Url <: BaseUrl] extends ScalatestMatchers with Eventually wi
     def sleep(duration: Duration): Unit = Page.SleepThread(duration)
   }
 
-  protected def waitUpTo(duration: Duration): PatienceConfig =
-    PatienceConfig(
-      // Wait up to 2 minutes for this operation
-      timeout = scaled(Span(AcceptanceSpecPatience.WAIT_SCALE * duration.toSeconds, Seconds)),
-      interval = scaled(Span(2, Seconds))
-    )
+  protected def waitUpTo(duration: Duration): PatienceConfig = PatienceConfig(
+    timeout = scaled(Span(AcceptanceSpecPatience.WAIT_SCALE * duration.toSeconds, Seconds)),
+    interval = scaled(Span(1, Seconds))
```

Comment:
should this be parameterized as well? I'm not sure how many places this is used but if it's used a lot, perhaps it could make things a bit slower.

Reply:
This guy is used in only a few special places when it's really needed.